### PR TITLE
BACK-40

### PR DESCRIPTION
**Pull request by Manizha-khorram**
Returing token at signUp route.

### CODE REVIEW

**Comment by larasodjati** [comment]
Good job @Manizha-khorram, I tested the sign-up route and it gave me a token when I created a user account. 
@jpremo could you please approve this PR? Thank you.

File: models/User.js [hunk 1]
```diff
@@ -49,18 +49,6 @@ const UserSchema = new mongoose.Schema({
...
-        required: [true],
-        min: -90,
-        max: 90,
-    },
-    longitude: {
-        type: Number,
-        required: [true],
-        min: -180,
-        max: 180,
-    },
```
Comment (jpremo) [inline models/User.js:63]: I don't think this should have been deleted.

**Review by jpremo — CHANGES_REQUESTED** [review]

File: models/User.js (hunk 1 above)
Comment (Manizha-khorram) [reply models/User.js:63]: > I don't think this should have been deleted.

It was extra Jacob, I pull the new updates. It doubled it! So I delete one of it!

Comment (jpremo) [reply models/User.js:63]: I see. Thanks for clarifying

**Review by jpremo — APPROVED** [review]

**Pull request merged**